delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/serialization/EventSerializer.scala

```diff
@@ -134,7 +134,7 @@ object EventSerializer {
   implicit final private val expandedDecoder: Decoder[ExpandedJsonLd] =
     Decoder.instance { hc =>
       for {
-        id       <- hc.up.get[Iri]("id")
+        id       <- hc.downArray.get[Iri](keywords.id) orElse hc.up.get[Iri]("id")
```

Comment:
is this to support the `@graph`?

Reply:
No. It is the normal use case how an expanded document looks like
```
[
   {
      "@id": "...",
      ...
   }
]
```